pkg/components/common/common.go

```diff
@@ -282,6 +290,107 @@ func (o *Operator) Uninstall(
 	return nil
 }
 
+func (o *Operator) DownloadKind(ctx context.Context) error {
+	return o.executor.DownloadKind()
+}
+
+func (o *Operator) CreateKindCluster(ctx context.Context) error {
+	cmd := "kind create cluster"
+	if _, _, err := o.executor.Exec(cmd); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (o *Operator) DeleteKind(ctx context.Context) error {
+	cmd := "kind delete cluster"
```

Comment:
If we agree to use `--name` to create the Kind cluster, then we also need to add the `--name` option when deleting it